_posts/2017-3-26-armory-and-hard-forks.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Armory and Hard Forks
+category: news
+date: 2017-03-26 23:30:00 -0500
+---
+
+Armory does not support any controversial hard forks such as Bitcoin Unlimited and will continue to support Segregated Witness and Bitcoin Core.
```

Comment:
How about changing this line to 

> The Armory developers (goatpig, droark, and achow101) oppose hard forks that may attack the original chain and support Segregated Witness and Bitcoin Core. Should a long-lasting hard fork that does not attack the original chain exist, we would implement functionality required to allow Armory users to transact on that chain only if absolutely necessary (e.g. the transaction format for that chain is changed).

The second sentence would require a bit of rewording.